File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/ui/toast/ToastModule.java
package com.androidcollider.easyfin.common.managers.ui.toast;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;

/**
 * @author Ihor Bilous
 */

@Module
public class ToastModule {

    @Provides
    @Singleton
    ToastManager provideToastManager() {
        return new ToastManager();
    }
}

File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/ui/toast/ToastModule.kt
package com.androidcollider.easyfin.common.managers.ui.toast

import dagger.Module
import dagger.Provides
import javax.inject.Singleton

/**
 * @author Ihor Bilous
 */
@Module
class ToastModule {
    @Provides
    @Singleton
    fun provideToastManager(): ToastManager {
        return ToastManager()
    }
}